lib/src/rules/email.ts

```diff
@@ -1,26 +1,15 @@
 import { FieldValidationResult, FieldValidationFunction } from '../entities';
+import { isValidPattern } from './pattern';
 
 // RegExp from http://emailregex.com
 const EMAIL_PATTERN = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 
 export const email: FieldValidationFunction = (value: string) => {
   const validationResult = new FieldValidationResult();
-  const isValid = isValidEmail(value);
+  const isValid = isValidPattern(value, EMAIL_PATTERN);
 
   validationResult.succeeded = isValid;
   validationResult.type = 'EMAIL';
   validationResult.errorMessage = isValid ? '' : 'Please enter a valid email address.';
```

Comment:
It would be a good idea for i18n. It could be public and be accesed via `Validators.messages.EMAIL_PATTERN` or something like that. I'll wait for some feedback.